config/default/capm3/namespace.yaml

```diff
@@ -2,3 +2,8 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: capm3-system
+  labels:
+    kubernetes.io/metadata.name: capm3-system
+    pod-security.kubernetes.io/enforce: restricted
+    pod-security.kubernetes.io/enforce-version: latest
```

Comment:
Same question as for the BMO PR, can we use the latest when we need to support much older k8s versions?

Reply:
It should work well in most cases AFAIK. The only mention of needing a specific version that I have seen is this: https://kubernetes.io/docs/concepts/security/pod-security-standards/#pod-os-field
> Note: Kubelets prior to v1.24 don't enforce the pod OS field, and if a cluster has nodes on versions earlier than v1.24 the restricted policies should be pinned to a version prior to v1.25.

So in case the management cluster is v1.24 or older this would need to be changed. Kubernetes v1.24 is EOL in ~2 months so I think it would be fine to keep it as is.

Reply:
OK, so if the `latest` is default, we can probably remove this line too.

Reply:
I didn't find anything in the documentation https://kubernetes.io/docs/concepts/security/pod-security-standards/#pod-os-field that this value is applied by default as the latest, so I'm not sure if I should remove this line. Even in the sample files given below they are providing the enforce-version.
https://raw.githubusercontent.com/kubernetes/website/main/content/en/examples/security/podsecurity-restricted.yaml

Reply:
For some reason they don't mention it in the [page for namespace labels](https://kubernetes.io/docs/tasks/configure-pod-container/enforce-standards-namespace-labels/). But [here](https://kubernetes.io/docs/concepts/security/pod-security-admission/) they mention that the version labels can be used to pin the version (so implies the default would be not pinned -> latest). In the page for configuring [cluster wide pod security standards](https://kubernetes.io/docs/tasks/configure-pod-container/enforce-standards-admission-controller/) they explicitly mention that latest is the default.

Reply:
I see, I'll update this as well since this makes much sense after going through these links.